cf-java-logging-support-core/beats/request-metrics/docs/fields.asciidoc

```diff
@@ -695,3 +695,132 @@ client, followed by proxy server addresses that forwarded the client
 request.
 
 
+==== x_forwarded_host
+
+type: string
+
+example: requested-host.example.com
+
+required: False
```

Comment:
What is the reason to have "False" starting with capital letter in this document?

Reply:
It is auto-generated. Always has been like that.